### ISSUE TITLE
where to set "predicting horizon"

### ISSUE THREAD

**Issue by wuyeyexvnainai** [ir-lab/Diff-Control#2]
when i use bcz I found a question:

Does this parameter represent how many steps to predict?

![Image](https://github.com/user-attachments/assets/bc3fbe57-47c7-4a5b-bdd0-d99397a8b545)

**Comment by yfzhoucs** [comment]
Hi,

Thanks for your question. Yes your are right, that is how many steps to predict.

**Comment by wuyeyexvnainai** [comment]
hello how many steps to predict is perfect？
and&nbsp;How much data is appropriate for training？


When I trained the model, the loss was extremely low, even reaching 10-6 power, but the performance of the validation set was poor. What is the reason for this


---Original---
From: "Yifan ***@***.***&gt;
Date: Mon, Feb 24, 2025 05:54 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ir-lab/Diff-Control] where to set "predicting horizon" (Issue#2)


   
Hi,
 
Thanks for your question. Yes your are right, that is how many steps to predict.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  yfzhoucs left a comment (ir-lab/Diff-Control#2)
 
Hi,
 
Thanks for your question. Yes your are right, that is how many steps to predict.

**Comment by liuxiao1468** [comment]
Hi, sorry for the late reply.

1. We have found the optimal prediction step to be 24 in our experiment.
2. We mentioned in the paper, for each task, we use 50 demos to train the model. 
3. It is okay for the validation loss to be high, please evaluate the model based of the succ rate after you trained.
